Formula/petsc.rb

```diff
@@ -0,0 +1,197 @@
+class Petsc < Formula
+  desc "Scalable solution of models that use partial differential equations"
+  homepage "https://www.mcs.anl.gov/petsc/index.html"
+  url "http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-lite-3.7.6.tar.gz"
+  sha256 "b07f7b4e57d75f982787bd8169f7b8debd5aee2477293da230ab6c80a52c6ef8"
+  revision 4
+  head "https://bitbucket.org/petsc/petsc", :using => :git
+
+  bottle :disable, "needs to be rebuilt with latest open-mpi"
```

Comment:
This should be removed. It was an artifact of the time when we stopped bottling in science.